### PR TITLE
Search Results Page (UI Only)

### LINKED ISSUE #50: Create search results page (UI only)
This ticket includes creating a basic UI for the search results page. It does NOT include backend search functionality or API calls - that work should be done under #44 

Results page should match the Figma mockup:
Screenshot 2024-09-11 at 10 20 49

### PR DESCRIPTION
**Pull request by katyasoup**
resolves #50 

- Adds styles for a search results landing page; all content is hard-coded
- Separates out SearchBar into its own component
- Adds Breadcrumb, SearchResults and SearchResultRow components

### CODE REVIEW

File: internal/ui/components/search_bar.templ [hunk 1]
```diff
@@ -0,0 +1,41 @@
+package components
+
+templ SearchBar() {
```
Comment (nickclyde) [inline internal/ui/components/search_bar.templ:3]: Thanks for moving this to a separate component!

File: internal/ui/components/breadcrumb.templ [hunk 2]
```diff
@@ -0,0 +1,15 @@
+package components
+
+templ Breadcrumb(currentPage string) {
```
Comment (nickclyde) [inline internal/ui/components/breadcrumb.templ:3]: 💯

**Review by nickclyde — APPROVED** [review]
Looks great! 🚀

**Review by akintner — APPROVED** [review]
this all looks great 🚀

**Pull request merged**